concordium-consensus/src/Concordium/GlobalState/Persistent/BlockState.hs

```diff
@@ -3311,24 +3322,33 @@ instance Cache.HasCache Modules.ModuleCache (PersistentBlockStateContext pv) whe
 instance (IsProtocolVersion pv) => MonadProtocolVersion (BlobStoreT (PersistentBlockStateContext pv) m) where
     type MPV (BlobStoreT (PersistentBlockStateContext pv) m) = pv
 
--- | Create a new account cache of the specified size for running the given monadic operation by
+-- | Create a new account cache of the specified size and a temporary 'LMDBAccountMap' for running the given monadic operation by
 --  extending the 'BlobStore' context to a 'PersistentBlockStateContext'.
-withNewAccountCache :: (MonadIO m) => Int -> BlobStoreT (PersistentBlockStateContext pv) m a -> BlobStoreT BlobStore m a
-withNewAccountCache size bsm = do
-    ac <- liftIO $ newAccountCache size
-    mc <- liftIO $ Modules.newModuleCache 100
-    alterBlobStoreT (\bs -> PersistentBlockStateContext bs ac mc) bsm
+-- Note. this function should only be used for tests.
```

Comment:
Because it adds a warning :D 

Thing is, it's used from the `KonsensusV1/TestMonad` so it'll produce a warning on each compilation. And I don't want to move stuff around too much in this PR.

Reply:
For things that are only for tests we often have warnings on the actual function, and disable those warnings in the test file only.